app/scripts/modules/titus/src/serverGroup/configure/wizard/pages/ServerGroupBasicSettings.tsx

```diff
@@ -19,10 +19,13 @@ import { DockerImageAndTagSelector, DockerImageUtils } from '@spinnaker/docker';
 import { ITitusServerGroupCommand } from '../../../configure/serverGroupConfiguration.service';
 
 const isNotExpressionLanguage = (field: string) => field && !field.includes('${');
-const isStackPattern = (stack: string) =>
-  isNotExpressionLanguage(stack) ? /^([a-zA-Z_0-9._${}]*(\${.+})*)*$/.test(stack) : true;
```

Comment:
@jrsquared PTAL I refactored these two regexps while fixing the lint errors

Reply:
lgtm